apps/utils/lib/helpers/normalize.ex

```diff
@@ -16,16 +16,36 @@ defmodule Utils.Helpers.Normalize do
   @moduledoc """
   Module to normalize strings.
   """
+  defmodule ToBooleanError do
+    defexception message: "Could not represent the value as boolean."
+    def error_message(value), do: "Could not represent the value (#{inspect(value)}) as boolean."
+  end
+
+  defmodule ToIntegerError do
+    defexception message: "Could not represent the value as an integer."
+    def error_message(value), do: "Could not represent the value (#{value}) as an integer."
+  end
 
   def string_to_boolean(<<"T", _::binary>>), do: true
   def string_to_boolean(<<"Y", _::binary>>), do: true
   def string_to_boolean(<<"t", _::binary>>), do: true
   def string_to_boolean(<<"y", _::binary>>), do: true
   def string_to_boolean(<<"1", _::binary>>), do: true
-  def string_to_boolean(_), do: false
+  def string_to_boolean(<<>>), do: nil
```

Comment:
are you sure this is what you'd expect? everything that starts with F,N,f,n or ) is considered `false`? what about "NYES"

Reply:
Though it applies the otherway with "YNO" with the current code too...

Anyhow, accio @sirn the original author. My thoughts back then is it's unlikely to happen, but since Ino raised....

Reply:
Can I answer YNO 😂

Reply:
The idea was that `string_to_boolean` is for reading ENV vars to be fed to somewhere that expects boolean, in which absent of value _usually_ means `false` (it's common to have `DISABLE_X=true` for those default-true operation), but I agree that absent of value should probably be `nil`.

I detect only first character because it's not uncommon to have value set to `X=no`, `X=yes`, `X=true`, `X=True`, `X=0`, `X=1`, `X=FALSE`… and the list goes on, and I'm not sure if I could cover all cases, so I did the whole first character thing...